### ISSUE TITLE
Latest Version of plugin 15 or 14?

### ISSUE THREAD

**Issue by magicprinc** [ebean-orm-tools/ebean-gradle-plugin#35]
https://plugins.gradle.org/plugin/io.ebean says Version 15.0.2 (latest)
but here I see only [14.0.2]

Is this OK?

What version should I use?

PS: Ebean lib itself has 15.1.0

**Comment by rbygrave** [comment]
> Is this OK?

Yes.  14.0.2 and 15.0.2 are exactly the same code, released twice with the 2 different versions.  

This is because at the moment we need to maintain both ebean 14.x and ebean 15.x versions (and these are almost the same except that 15.x has some features removed from it).

TLDR: This is ok, 14.x and 15.x versions of ebean-agent, ebean-maven-plugin, ebean-grade-plugin are all the same.

**Comment by magicprinc** [comment]
OK, now I understand why

with the latest ebean-gradle-plugin
`Transforming ebean-gradle-plugin-15.0.2.jar (io.ebean:ebean-gradle-plugin:15.0.2) with ExternalDependencyInstrumentingArtifactTransform`

I see 
`Transforming ebean-agent-14.0.2.jar (io.ebean:ebean-agent:14.0.2) with ExternalDependencyInstrumentingArtifactTransform`

Although it is a little misleading. 

PS: Could you upgrade agent version in Gradle plugin and IDEA plugin, when you have time please 🙏
